browser/ui/views/brave_actions/brave_actions_container.cc

```diff
@@ -42,6 +44,15 @@
 #include "ui/views/layout/grid_layout.h"
 #include "ui/views/view.h"
 
+namespace {
+
+bool UsingRewardsExtension() {
```

Comment:
If you're going to use a helper method like this, it's better to put it in features.cc/.h and use it consistently

Reply:
I'll remove it.